app/view_objects/assessor_interface/application_forms_index_view_object.rb

```diff
@@ -46,6 +46,7 @@ def status_filter_options
       awarded
       declined
       potential_duplicate_in_dqt
+      assessment_in_progress
```

Comment:
Adding this here means we'll end up with two "Assessment in progress" entries in the filter which would be confusing for assessors, I think instead we should replace the `initial_assessment` entry with `assessment_in_progress` and make sure the status filter will handle that correctly.